tests/test_configure_feedstock.py

```diff
@@ -545,3 +545,21 @@ def test_files_skip_render(render_skipped_recipe, jinja_env):
     for f in skipped_files:
         fpath = os.path.join(render_skipped_recipe.recipe, f)
         assert not os.path.exists(fpath)
+
+
+def test_automerge_action(py_recipe, jinja_env):
+    cnfgr_fdstk.render_actions(
+        jinja_env=jinja_env,
+        forge_config=py_recipe.config,
+        forge_dir=py_recipe.recipe,
+    )
+    cnfgr_fdstk.copy_feedstock_content(py_recipe.config, py_recipe.recipe)
+    assert os.path.exists(
+        os.path.join(py_recipe.recipe, ".github/workflows/main.yml")
```

Comment:
Can you also add a test that this file does not exist by default?

Reply:
done!